src/main/java/cn/nukkit/network/protocol/ToastRequestPacket.java

```diff
@@ -0,0 +1,28 @@
+package cn.nukkit.network.protocol;
+
+import lombok.ToString;
+
+@ToString
+public class ToastRequestPacket extends DataPacket {
+
+    public String mTitle = "";
+    public String mContent = "";
```

Comment:
You probably should change these variables to just title and content. Its not C++.